test/acceptance/integration/steps_lib/mock_utils.py

```diff
@@ -71,7 +71,6 @@ def the_path_in_the_last_petition_contains(step, head, value):
     mock_response = eval(world.mock_data.text)['requests']
```

Comment:
Its not necessary, but i will study the use of constante in the next implementation
NTC